docs/conf.py

```diff
@@ -62,35 +76,36 @@
 
 from sunpy import version as versionmod
 
-# -- Shut up numpy warnings from WCSAxes
+# -- Shut up numpy warnings from WCSAxes --------------------------------------
 import numpy as np
 np.seterr(invalid='ignore')
 
 # -- Download Sample Data -----------------------------------------------------
-
 import sunpy.data.sample
 
 # -- General configuration ----------------------------------------------------
 
 # If your documentation needs a minimal Sphinx version, state it here.
-needs_sphinx = '1.5'
+needs_sphinx = '1.6'
 
 # To perform a Sphinx version check that needs to be more specific than
 # major.minor, call `check_sphinx_version("x.y.z")` here.
 check_sphinx_version(needs_sphinx)
 
-# The intersphinx_mapping in astropy_helpers.sphinx.conf refers to astropy for
-# the benefit of affiliated packages who want to refer to objects in the
-# astropy core.  However, we don't want to cyclically reference astropy in its
-# own build so we remove it here.
-#del intersphinx_mapping['astropy']
-
-# add any custom intersphinx for sunpy
+# Add any custom intersphinx for SunPy
 intersphinx_mapping.pop('h5py', None)
 intersphinx_mapping['sqlalchemy'] = ('http://docs.sqlalchemy.org/en/latest/', None)
 intersphinx_mapping['pandas'] = ('http://pandas.pydata.org/pandas-docs/stable/', None)
 intersphinx_mapping['skimage'] = ('http://scikit-image.org/docs/stable/', None)
 
+# Load data about stability
+with open('./dev_guide/sunpy_stability.yaml', 'r') as estability:
+    sunpy_modules = yaml.load(estability.read(), Loader=yaml.Loader)
```

Comment:
this should probably be in the try bit

Reply:
I think it should either be in the function or in the `if` block at the end.